aggregator/src/aggregator.rs

```diff
@@ -1297,6 +1297,18 @@ impl VdafOps {
             ));
         }
 
+        // Reject reports that would be eligible for garbage collection, to prevent replay attacks.
+        if let Some(report_expiry_age) = task.report_expiry_age() {
+            let report_expiry_time = report.metadata().time().add(report_expiry_age)?;
+            if clock.now().is_after(&report_expiry_time) {
```

Comment:
We should ideally sample `clock.now()` once, to avoid weird "impossible" bugs around things right at the edge.